torchrl/trainers/helpers/models.py

```diff
@@ -1351,3 +1351,5 @@ class DiscreteModelConfig:
     # whether a distributional loss should be used.
     atoms: int = 51
     # number of atoms used for the distributional loss
+    gSDE: bool = False
```

Comment:
Yeah let's not do that :) No gSDE for discrete actions

Reply:
https://github.com/facebookresearch/rl/pull/201/commits/03b8c78b7c39cfd40d4d927d08339ce6fe70fc2c

Reply:
I removed epsilon greedy wrapper which removed the error with action_value being None